source/pidomus.cc

```diff
@@ -871,7 +896,7 @@ shared_ptr<typename LAC::VectorType>
 piDoMUS<dim, spacedim, LAC>::create_new_vector() const
 {
   shared_ptr<typename LAC::VectorType> ret = SP(new typename LAC::VectorType(solution));
-  *ret *= 0;
+  //   *ret *= 0;
```

Comment:
`create_new_vector()` is called from imex/ida/kinsol. It is supposed to overwrite this vector, so there **should** be no difference. I comment and uncomment it to double check that there is no difference between the two cases. Tell me what do you prefer